test/solveEquation/solveEquation.test.js

```diff
@@ -128,3 +128,21 @@ describe('solveEquation errors', function() {
   ];
   tests.forEach(t => testEquationError(t[0], t[1]));
 });
+
+function testPerformTermOperationExpressionStep(equationString, outputString, step, debug=false) {
```

Comment:
oo I like this idea of testing steps like this - wonder if we should use it elsewhere

it might break if we change how we apply steps, but we can always come back and change the tests if so, I think this is the best way to do it :) nice!